project-geometry/src/main/java/com/esri/arcgismaps/sample/projectgeometry/MainActivity.kt

```diff
@@ -91,11 +91,13 @@ class MainActivity : AppCompatActivity() {
         // required to access basemaps and other location services
         ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
         lifecycle.addObserver(mapView)
-
+        // create and add a map with a navigation night basemap style
+        val map = ArcGISMap(BasemapStyle.ArcGISNavigationNight)
         // configure mapView assignments
         mapView.apply {
-            // create and add a map with a navigation night basemap style
-            map = ArcGISMap(BasemapStyle.ArcGISNavigationNight)
+            this.map = map
+            // add our marker overlay to the graphics overlay
+            graphicsOverlay.graphics.add(markerGraphic)
```

Comment:
This change makes sense to me but just want to double-check what the behaviour is when the marker graphic has no geometry (i.e. before `onGeoViewTapped` first runs). Does it still appear on the screen or is it invisible?

Reply:
Yeah it doesn't appear on the MapView, since the geometry property is initially null.